metasynth/dataset.py

```diff
@@ -229,6 +229,21 @@ def to_json(self, fp: Union[pathlib.Path, str],
         with open(fp, "w", encoding="utf-8") as f:
             json.dump(self_dict, f, indent=4)
 
+    def export(self, fp: Union[pathlib.Path, str],
+               validate: bool = True) -> None:
+        """Serialize and export the MetaSynth dataset to a JSON file.
+
+        Optional validation against a JSON schema included in the package.
+
+        Parameters
+        ----------
+        fp:
+            File to write the dataset to.
+        validate:
+            Validate the JSON file with a schema.
+        """
+        self.to_json(fp, validate)
```

Comment:
I think if we want to talk about the `export` in the documentation (instead of the `to_json` file), then we should also put the implementation here, and make the `to_json` method the wrapper function.